docs/permission_template.md

```diff
@@ -0,0 +1,15 @@
+# Permission Template
+
+The permission template is a mechanism in sonar to setup project permissions. The default template will be changed
+during the dogu startup to ensure that the CES_ADMIN group has access to administer new created projects. The following
+permissions will be set (admin codeviewer issueadmin securityhotspotadmin scan user ) this setup can be verified
+(Administration -> Security -> Permisssion Templates). *see setup.json for further details*
+
+![default template overview][assets/default_template_ces_admin_permissions.png]
+
+Apart from project permissions there are some global permission, these permission will be set up during
+start up as well. Defined global permissions for the CES_ADMIN group will be (admin profileadmin gateadmin provisioning)
+
+The permissions for projects created without correct CES_ADMIN group permissions can be changed later using a specific etdkey.
```

Comment:
Naming/missing info:

## Naming
> etdkey
I it is more encouraged to say "registry key" or "config key" because maybe the DB implementation can change.

## Missing info

Also I miss any mention that this is some sort of one-time-toggle. IIRC the logic goes like this:
1. the key will be configured by an administrator
2. on start-up the key will be read to trigger a change in the project templates and permission grantings
3. the key will be reset after permissions were successfully granted

The reason for this behaviour (in contrast to always reset permission templates and apply them) is that this is a very time consuming action (a comment below says ~2 secs per project). Administrators with a SonarQube instance with very large project numbers should be aware of the duration.

Reply:
I think `config key` fits well. I am aware of that mechanism, it is already documented directly at the key description. I can put it inside the documentation if you insist. Fun Fact: I wrote a little script to create a bunch of projects in the previous sonar (~50) than migrated the admin group, it only took around (8 sec for all projects). Im not sure where the measurement 2 sec per project are coming from.